Gate_system/Models/ReportItem.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
Megsti palikti daug nenaudojamu usings, istrink. Taip pat komitinai i repozitorija pdf faila su uzduotimi, to tikrai nereiktu komitinti.